uavcan_bridge_nodelet:
    uavcan_id: 10
